SecurityManager/Clients.Web/res/Themes/NovaGray/Html/Style.css

```diff
@@ -259,4 +265,10 @@ td.buttonCell span.DropDownMenuSelect
 td.buttonCell span.DropDownMenuSelect span.DropDownMenuButton
 {
   left: 0;
+}
+
+table.bocListTable tbody.bocListTableBody td.bocListDataCell a img.Icon
```

Comment:
It would rather be `img.Icon[src$=#MyImage]`, but yes, it would work.

The above CSS is for the following element:
![image](https://user-images.githubusercontent.com/40357550/128319992-623b90e4-6e54-46a7-a94f-88117e4d1586.png)

Just tell me how I should simplify it, or if I should change it to the version you mentioned, that would be alright.

Reply:
Ah okay. So, there's two different places that use Icon.
one in the BocListDataCell, the other in the conditionCell.
![image](https://user-images.githubusercontent.com/6972548/128326385-f8ca17ac-21a9-484a-99dd-d777ccfa19e5.png)

The Icon in the BocListDataCell will likely have a predefined default size evntually but for now, this works. and yes, simplify please. we don't need the whole path for this.

Reply:
Changed it in [`f6f9bf3` (#160)](https://github.com/re-motion/Framework/pull/160/commits/f6f9bf379d7c922c610d27d949e0021fd8eac5cd), please also review the change removing `.buttonCell`, to make the rule apply to `.conditionCell` too.